book/website/profiles.yml

```diff
@@ -0,0 +1,58 @@
+- name: Data Study Group
+  files:
+    - reproducible-research/reproducible-research
+    - reproducible-research/overview
+    - reproducible-research/overview/overview-resources
+    - reproducible-research/overview/overview-barriers
+    - communication/communication
+    - communication/comms-overview
+    - communication/comms-overview/comms-overview-principles
+    - collaboration/collaboration
+    - collaboration/github-novice
+    - collaboration/github-novice/github-novice-advanced
+    - collaboration/leadership
+  colour: white
+  description: consectetur adipiscing elit. Sed nec mauris quis magna consequat consequat vel quis odio. Sed nec arcu malesuada, aliquam erat eget, bibendum nisi. Nunc varius, odio eu molestie euismod, sapien ipsum vulputate eros, sed lacinia nisl sapien eu nisi. Nulla facilisi. Nullam vulputate, odio sed tempor rutrum, est ipsum ultrices lorem, a feugiat velit magna in metus.
```

Comment:
Lorem ipsum?

Reply:
This will be updated -- in fact, I have a local file that needs to be pushed here.

Reply:
Updated the file.